doc/RecordingControl.xml

```diff
@@ -1684,8 +1766,220 @@ SetRecordingJobMode(JobToken, Active)
+        The device shall replace <literal>date</literal> and <literal>time</literal> respectively with the date and time portion of the span end in Universal Coordinated Time (UTC), following the ABNF rules.
+        The device shall replace <literal>counter</literal> by a value equal to the counter of the last segments of the span plus 1.
+        The device shall replace <literal>extension</literal> with a value defined by the recording formats in the following sections.
+      </para>
+    </section>
+    <section>
+      <title>Other objects</title>
+      <para>
+        The device may store additional objects with object keys starting with <literal>lead "/"</literal> according to the ABNF rules and replacement rules defined in <xref linkend="_refCloudRecordingSegmentObjects" />, as long as their object keys do not end in an extension defined in this annex.
+      </para>
+    </section>
+    <section>
+      <title>Recording format MP4</title>
+      <para>
+        For the recording format <literal>MP4</literal>, the recording information shall be written as multiplexed tracks into a single segment.
+        Each segment shall contain zero or one tracks with video data, zero or one tracks with audio data, and zero or one tracks with metadata.
+        The device shall generate the object key for each segment with the <literal>extension</literal> <literal>"mp4"</literal>.
+      </para>
+      <para>
+        The device shall upload each segment object with the content type <literal>video/mp4</literal>.
+        The device should include the <literal>codecs</literal> parameter in the content type according to RFC 6381.
+      </para>
+      <para>
+        The <literal>MovieBox</literal> shall not reference any samples.
+      </para>
+      <para>
+        A segment shall contain a <literal>MovieFragmentRandomAccessBox</literal> as the last box.
+        The <literal>MovieFragmentRandomAccessBox</literal> shall provide references for all tracks and their applicable <literal>MovieFragmentBox</literal> instances.
+      </para>
+      <para>
+        The device shall upload a span end object.
+        It should generate the object key for a span object with the <literal>"mp4_end"</literal> as the <literal>extension</literal>.
+      </para>
+    </section>
+    <section>
+      <title>Recording format CMAF</title>
+      <para>
+        For the recording format <literal>CMAF</literal>, the recording information shall be written as as one track per segment.
+        Each segment either contains one track with video data, one track with audio data, or one track with metadata, indicated by the extension.
+        When closing multiple types of segments of an underlying span at the same time, the device shall only increment the segment counter by 1.
```

Comment:
Can we also say that each media track has its own sequence numbers and the segments don't need to align (ex: audio files could be longer than the video files) ?

Reply:
I updated the notion of a span, this is now true by definition.